### ISSUE TITLE
Multiple front ends hosting with the same cognito

### ISSUE THREAD

**Issue by robin22d** [aws-amplify/amplify-cli#6282]
I created an aws amplify back end  as follows:

```
amplify init
```
```
amplify add auth
```
```
amplify push
```
```
amplify publish
```

Once this was done I created a front end and used the amplify pull command in the front end app

```
amplify pull --appId ************* --envName *******
```

I wanted to host this app so in the front end app I did

```
amplify init
```
```
amplify add hosting
```
```
amplify push
```
```
amplify publish
```

For the other front ends that i wanted to use the same backend on i created them in the same way and this works fine.

Is this the correct way to do this if so if i make changes to the cognito how would i get the updates to show on the front end?

**Comment by jhockett** [comment]
Hi @robin22d, check out the [docs](https://docs.amplify.aws/cli/teams/multi-frontend) we have for multiple frontends.

**Comment by robin22d** [comment]
Hi yes I have had a look at the docs but I cannot see anything about hosting unless im missing something? Also do you know of any docs that would help me with when a user is logged into one how would I make is so they dont have to log into the app front ends?

**Comment by jhockett** [comment]
Is there something not covered in the [hosting](https://docs.amplify.aws/cli/hosting) section? What are you trying to accomplish?

> Also do you know of any docs that would help me with when a user is logged into one how would I make is so they dont have to log into the app front ends?

I don't fully understand the question. How would the user log in before accessing the FE?

**Comment by robin22d** [comment]
> I don't fully understand the question. How would the user log in before accessing the FE?

I have two apps 

* app 1
* app 2

The user has access to both apps and they log into app 1. Then when they go to app 2 they should already be logged in because both apps have the same back end. Its like login into gmail and then going to google docs and not having to log in again.

**Comment by robin22d** [comment]
> Is there something not covered in the [hosting](https://docs.amplify.aws/cli/hosting) section? What are you trying to accomplish?

Yes I have had a look that the hosting and wanted to know if the way that im doing it is correct.

**Comment by jhockett** [comment]
Hi @robin22d, thanks for clarifying! It sounds to me like you are looking for SSO ([Single sign-on](https://en.wikipedia.org/wiki/Single_sign-on)). Amplify does not currently have an out-of-the-box solution for SSO. I've marked this issue as a feature request.
